Dockerfile

```diff
@@ -0,0 +1,13 @@
+# Stage 0, "build-stage", based on Node.js, to build and compile the frontend
+FROM tiangolo/node-frontend:10 as build-stage
+WORKDIR /app
+COPY package*.json /app/
+RUN npm install
+COPY ./ /app/
+RUN npm run build
+# Stage 1, based on Nginx, to have only the compiled app, ready for production with Nginx
+FROM nginx:1.15

```

Comment:
```suggestion
FROM nginx:latest
```

Reply:
It's good practice to lock the code on a specific version and perform periodical upgrades when **needed**. If "**latest**" is always used, then the image is vulnerable to breaking if the latest push from nginx is not stable.
May I suggest **nginx:1.17.6-alpine** it's currently the latest and the smallest in size.

Reply:
i agree on that.